tools/wpt/run.py

```diff
@@ -706,7 +706,7 @@ def setup_logging(kwargs, default_config=None, formatter_defaults=None):
 def setup_wptrunner(venv, prompt=True, install_browser=False, **kwargs):
     from wptrunner import wptcommandline
 
-    kwargs = utils.Kwargs(kwargs.iteritems())
+    kwargs = utils.Kwargs(kwargs.items())
```

Comment:
As a point of principle let's use `six.iteritems(kwargs)` here.

Reply:
OK, I thought the idea was to use six only if strictly needed. Will provide a new patch.

Reply:
I don't recall that being a rule, but to the extent it is, I think "strictly needed" would cover cases where there's a semantic change from the non-six alternative (e.g. eagerly evaluating an iterator). As you say for this case it likely doesn't matter, but in general I'd prefer to avoid cases where we change the Python 2 behaviour or have gratuitous 2/3 differences, since those make it harder to reason about the code.